recipes/libdivide/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,8 @@
+#include <libdivide.h>
+
+#include <cstdint>
+
+int main() {
+  libdivide::divider<int64_t> fast_d(30);
```

Comment:
Could we check for example, that 60 divided by 30 is 2 to make the test just a bit more reliable?

Reply:
test_package is not a unit test, but yes it sounds reasonable ;)

Reply:
I've added a display of a division result, but no check.